services/Resources/subscriptions/Deploy-ActivityLog-ServiceHealth-Incident.json

```diff
@@ -215,22 +207,21 @@
                   "enabled": {
                     "type": "string"
                   },
-                  "ALZMonitorActionGroupEmail": {
-                    "type": "string"
+                  "ALZMonitorActionGroupEmails": {
+                    "type": "Array"
                   },
-                  "BYOActionGroup": {
-                    "type": "String"
+                  "BYOActionGroups": {
+                    "type": "Array"
                   }
                 },
                 "variables": {
-                  "varBYOActionGroups": "[[split(parameters('BYOActionGroup'), ',')]",
-                  "copy": [
+                  "copyLoop": [
                     {
-                      "name": "varActionGroupIds",
-                      "count": "[[length(variables('varBYOActionGroups'))]",
+                      "name": "actionGroups",
+                      "count": "[[[length(parameters('BYOActionGroups'))]",
                       "mode": "serial",
                       "input": {
-                        "actionGroupId": "[[trim(variables('varBYOActionGroups')[copyIndex('varActionGroupIds')])]"
+                        "actionGroupId": "[[[trim(parameters('BYOActionGroups')[copyIndex('actionGroups')])]"
```

Comment:
This is because we move the copy logic into another deployment scope, in order for arm schema to accept it, otherwise it will try to resolved the copy logic before needing to

Reply:
Got it. I recognize the change but haven't had the chance to evaluate it thoroughly. I'll test and review the changes, and pending any additional feedback, I'll merge them into the development branch. Thanks!